WLC/image_processing/character.py

```diff
@@ -9,17 +9,20 @@
 
 
 class Character(ExtendedImage):
-    def __init__(self, image, x, y, w, h):
-        super().__init__(image, x, y, w, h)
+    def __init__(self, image, x_axis, y_axis, width, height, extended_image=None,
+                 show_pic=False, show_line=False, show_word=False, show_char=False):
+        super().__init__(image, x_axis, y_axis, width, height, extended_image,
+                         show_pic, show_line, show_word, show_char)
         self.ocr = OCR()
 
+        if self.show_char:
+            cv2.imshow("Character", image)
+            cv2.waitKey(0)
+
     def get_code(self):
-        # TODO: plugin to tensorflow
+        # TODO: plugin to Tensorflow/Keras
```

Comment:
We have already plugged Keras in this function. I think we should remove this before we this MR is accepted.